ansible/roles/set-repositories/tasks/rhn-repos.yml

```diff
@@ -23,6 +23,7 @@
     password: "{{ rhel_subscription_pass }}"
     pool_ids: "{{ rhsm_pool_ids }}"
     auto_attach: false
+  ignore_errors: true  # Because of SCA
```

Comment:
we usually don't ignore_errors without a strong justification, especially with a role used by other configs and demos.

Can you explain more thoroughly what this is needed?

Reply:
During development phase, I was using the RHN repositories and my account had the Simple Content Access enabled, so when running these two modules, the attachment to a valid subscription wasn't successful and the task was failing. I decided to add the `ignore_errors: true` because it was easier than adding a complete and correct `changed_when` or `failed_when`.

Reply:
if you don't plan to fix the case you're describing (using your account), then please remove the ignore_errors line as this is going to production and won't be helpful there.